src/sidebar/templates/annotation-thread.html

```diff
@@ -4,8 +4,8 @@
        ng-class="vm.threadToggleClasses()"
        title="{{vm.thread.collapsed && 'Expand' || 'Collapse'}}"
        ng-click="vm.toggleCollapsed()">
-       <span ng-class="{'h-icon-arrow-right': vm.thread.collapsed,
-                        'h-icon-arrow-drop-down': !vm.thread.collapsed}"></span>
+       <svg-icon name="'caret-right'" ng-if="vm.thread.collapsed"></svg-icon>
+       <svg-icon name="'expand-menu'" ng-if="!vm.thread.collapsed"></svg-icon>
```

Comment:
Now that I look at it, the conditional could just be put in the value of the `name` prop:

```
<svg-icon name='vm.thread.collapsed ? 'caret-right' : 'expand-menu'></svg-icon>
```

This will be a little more efficient, although I don't think it will make any user-visible difference.